Source/m_menu.c

```diff
@@ -3094,10 +3098,7 @@ static const char *weapon_attack_alignment_strings[] = {
 static void M_UpdateCenteredWeaponItem(void)
 {
   // weap_center
-  if (cosmetic_bobbing)
-      weap_settings1[12].m_flags &= ~S_DISABLE;
-  else
-      weap_settings1[12].m_flags |= S_DISABLE;
+  DISABLE_ITEM(!cosmetic_bobbing, weap_settings1[13]);
```

Comment:
`[12]` or `[13]`?

Reply:
`[13]`, you can turn off bobbing and enable recoil pitch, but not center weapon. Not sure why, maybe we should revist that code.